components/posts/posts.php

```diff
@@ -132,25 +148,23 @@ function mm_posts( $args ) {
 
 	ob_start(); ?>
 
-	<div class="<?php echo esc_attr( $mm_classes ); ?>">
+	<?php do_action( 'mm_posts_before', $query, $context, $args ); ?>
```

Comment:
Or `mm_posts_before_loop` is fine if you add `mm_posts_after_loop`, instead of `_inside`